.golangci.yml

```diff
@@ -62,6 +62,9 @@ linters-settings:
 
 issues:
   exclude-rules:
+    - path: fpga_plugin\.go
+      linters:
+        - gosec
```

Comment:
@tkatila Just out of curiosity, what does gosec complain about?

Reply:
It's about the permissions for the CDI file creation here: https://github.com/intel/intel-device-plugins-for-kubernetes/pull/1810/files#diff-26ecd1222eba9f1867293a1f1a2c233c98eaa9d24f0e4254a319f0d1ea397ee2R408

Reply:
I had forgotten that I can also ignore a lint per line. I'll change it.